lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -922,8 +922,7 @@ def bind_expression(self, bindvalue):
 from sqlalchemy.sql import elements
 from ... import sql, schema, exc, util
 from ...engine import default, reflection
-from ...sql import compiler, expression
-from ...sql import sqltypes
+from ...sql import compiler, expression, selectable, sqltypes
```

Comment:
OK this is way older than current master as the imports and such have been reformatted some weeks ago, can you re-push against current master?